projects/packages/backup-helper-script-manager/tests/php/test-class-throw-on-errors.php

```diff
@@ -0,0 +1,504 @@
+<?php // phpcs:disable WordPress.Files.FileName.InvalidClassFileName
+
+// phpcs:disable Generic.Commenting.DocComment.MissingShort
+// phpcs:disable Squiz.Commenting.FileComment.Missing
+// phpcs:disable Squiz.Commenting.FunctionComment.EmptyThrows
```

Comment:
You could do like `projects/packages/my-jetpack/tests/php/.phpcs.dir.xml` instead of having to manually exclude a bunch of stuff in the tests.

Reply:
I want to ignore those in just a single test file (but not the other), so I think it's more KISS that way.